index.html

```diff
@@ -207,8 +207,8 @@ <h2>Download</h2>
         </section>
         <section id="community">
             <h2>Community and contribution</h2>
-            <p>In case you have any questions or just simply want to chat with other muCommander users, please join the community on our Gitter channel.</p>
-            <p>If you would like to help improve muCommander, please join the development on GitHub. There are many ways how you can help, other than writing code.
+            <p>In case you have any questions or just simply want to chat with other muCommander users, please join the community on our <a href="https://gitter.im/mucommander" target="_blank" rel="nofollow">Gitter channel</a>.</p>
```

Comment:
I get 404 for that link but honestly, I don't track Gitter anymore.. I think we shall move to Github discussions at this point - otherwise there are too many things to track

Reply:
> I get 404 for that link but honestly, I don't track Gitter anymore.. I think we shall move to Github discussions at this point - otherwise there are too many things to track

oops, the correct address is https://gitter.im/mucommander/Lobby - so maybe let's close this PR and I will provide PR replacing Gitter with github discussion link